Dropdown/src/Dropdown.jsx

```diff
@@ -412,7 +415,8 @@ Dropdown.defaultProps = {
     autoHide: true,
     className: "",
     isDropDownOpen: false,
-    selectedIndex: -1
+    selectedIndex: -1,
+    decode:(label) => label
```

Comment:
I'm not sure about the name `decode`. Maybe it could be more descriptive. Like `getSearchText`?